pkg/operator/ceph/cluster/mon/health_test.go

```diff
@@ -380,82 +377,6 @@ func TestAddOrRemoveExternalMonitor(t *testing.T) {
 	assert.Equal(t, 2, len(c.ClusterInfo.Monitors))
 }
 
-func TestForceDeleteFailedMon(t *testing.T) {
```

Comment:
There is a unit test already covering the scenarios in the mon and osd unit tests?

Reply:
Do you mean we need these test scenarios even we don't have the respective functions?

Reply:
Since we're deleting this unit test, I just wanted to confirm the force deleting of pods is still covered by a separate unit test.

Reply:
yes, force deleting of pods is covered in the unit tests for the new code path